src/styles/variables.scss

```diff
@@ -0,0 +1,12 @@
+$icon-size-mob: 24px;
+$icon-size-tablet: 32px;
+$menu-bg-color: #e5e6e8;
+$main-text-color: #292929;
+$button-text-color: #fff;
+$black-color: #111;
+$button-height: 56px;
+$bg-grey-color: #f8f8fa;
+$accent-color: #f0f1f4;
+
+$duration: 0.3s;
+$scale: 1.1;
```

Comment:
please pick more informant names for variables

ex: `animation-duration`, `image-hover-scale`, etc